code/cgame/cg_spraylogo.c

```diff
@@ -474,10 +474,7 @@ void Add_LogoToDrawList(const vec3_t origin, vec3_t dir, qhandle_t shader, float
 		lp->spraytime = cg.time;
 		lp->logoShader = shader;
 		lp->numVerts = lf->numPoints;
-		lp->color[0] = (float)color[0] / 255.0f;
-		lp->color[1] = (float)color[1] / 255.0f;
-		lp->color[2] = (float)color[2] / 255.0f;
-		lp->color[3] = (float)color[3] / 255.0f;
+		Vector4Copy(color, lp->color);
```

Comment:
this won't work - the color array here is of type byte, not float - and as such the values are in the range 0-255, for float color values they are normalized in the range 0.0-1.0. The dividing here is still needed.

Reply:
ah you also changed from byte to float - i missed that type change...